index.bs

```diff
@@ -139,6 +139,120 @@ The terms [=fulfilled=], [=rejected=], [=resolved=], [=pending=] and
 The terms {{ReadableStream}} and {{WritableStream}} are defined in
 [[!WHATWG-STREAMS]].
 
+# Protocol concepts # {#protocol-concepts}
+
+A <dfn>WebTransport session</dfn> is a session of WebTransport over HTTP/3.
+There may be multiple [=WebTransport session=]s on one [=connection=], when pooling is enabled.
+
+[=WebTransport session=] has the following capabilities defined in [[!WEB-TRANSPORT-HTTP3]].
+
+<table class="data" dfn-for="session">
+ <thead>
+  <tr>
+   <th>capability
+   <th>definition
+  </tr>
+ </thead>
+ <tbody>
+  <tr>
+   <td><dfn>send a datagram</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.4](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.4)
+  </tr>
+  <tr>
+   <td><dfn>receive a datagram</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.4](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.4)
+  </tr>
+  <tr>
+   <td><dfn>create an [=stream/outgoing=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.1](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.1)
+  </tr>
+  <tr>
+   <td><dfn>create a [=stream/bidirectional=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.2](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.2)
+  </tr>
+  <tr>
+   <td><dfn>receive an [=stream/incoming=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.1](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.1)
+  </tr>
+  <tr>
+   <td><dfn>receive a [=stream/bidirectional=] stream</dfn>
+   <td>[[!WEB-TRANSPORT-HTTP3]]
+   [section 4.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.3)
+  </tr>
+ </tbody>
+</table>
+
+To <dfn for=session>establish</dfn> a [=WebTransport session=], follow [[!WEB-TRANSPORT-HTTP3]]
+[section 3.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-3.3).
+
+To <dfn for=session>terminate</dfn> a [=WebTransport session=] |session| with |code|, run these
+steps:
+
+1. Terminate |session|, as described in [[!WEB-TRANSPORT-HTTP3]]
+   [section 5](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-5),
+   with using |code| as each RESET_STREAM frame's "Application Protocol Error Code"
```

Comment:
Could you remove the specific mention of RESET_STREAM?  We may change the way how this works soon ietf-wg-webtrans/draft-ietf-webtrans-http3#41.

Reply:
Done. Can you mention the "code" parameter in https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-5?

Reply:
I'll add it when I rewrite it to use trailers.